docs/UserGuide.md

```diff
@@ -75,26 +75,46 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a volunteer: `vadd` [COMING SOON]
```

Comment:
could probably make it clearer: "Adding a volunteer to an event" but given the description already covers it i don't think it matters much for now (dioni might kill us)

Reply:
yea agree